### PR TITLE
Onboarding tour accessibility fixes, animation improvement

### PR DESCRIPTION
**Pull request by kimprice**
This PR improves:
- animation by making it faster and including width and height in transitions
- visibility of tabbing / focus on buttons (closes https://github.com/microsoft/pxt-microbit/issues/5012)
- screen reading (closes https://github.com/microsoft/pxt-microbit/issues/5014)
  - tested with Narrator and NVDA
  - it's not perfect, but it should read all the content now

Try it out: https://makecode.microbit.org/app/ad2675c5dac61c2bed8b9b8816708b290fda6d92-5997b1ced2

tabButton

### CODE REVIEW

File: react-common/styles/onboarding/TeachingBubble.less [hunk 1]
```diff
@@ -32,6 +32,16 @@
     border-radius: 0.5rem;
     padding: 1rem;
     z-index: @modalDimmerZIndex;
+
+    .common-button {
+
+        &:focus {
```
Comment (jwunderl) [inline react-common/styles/onboarding/TeachingBubble.less:38]: can we make this darken the button a bit as well? It's pretty subtle with just a small extra outline, especially on the white button

File: react-common/styles/onboarding/TeachingBubble.less [hunk 2]
```diff
@@ -32,6 +32,16 @@
...
+            outline: .1rem solid @white;
+        }
+        &:focus::after {
```
Comment (jwunderl) [inline react-common/styles/onboarding/TeachingBubble.less:41]: do we normally set outline using an ::after pseudoelement that this is overriding?

**Review by jwunderl — APPROVED** [review]

**Review by thsparks — APPROVED** [review]
I agree with Joey's comment about darkening the button (or something similar to make button selection more apparent), otherwise LGTM!

**Review by eanders-ms — APPROVED** [review]

File: react-common/styles/onboarding/TeachingBubble.less (hunk 2 above)
Comment (kimprice) [reply react-common/styles/onboarding/TeachingBubble.less:41]: Found it. I think it's this: https://github.com/microsoft/pxt/blob/master/react-common/styles/controls/Button.less#L158

**Comment by kimprice** [comment]
> Not blocking, but at 125% zoom, I see a thin white border strike across the bubble tab: image

Yea, I've noticed this happens on smaller screens, too. I'll file an issue for it.

File: react-common/styles/onboarding/TeachingBubble.less (hunk 1 above)
Comment (kimprice) [reply react-common/styles/onboarding/TeachingBubble.less:38]: @jwunderl @thsparks our buttons generally only darken on hover, do you want me to add this effect for on focus for all buttons or just for the teaching bubble?

Comment (kimprice) [reply react-common/styles/onboarding/TeachingBubble.less:38]: I did it for the teaching bubble only, but using the same darkening method we use for on hover:
tabButton2

Comment (kimprice) [reply react-common/styles/onboarding/TeachingBubble.less:38]: I'm going to merge, but feel free to file an issue if I should make any other change.

**Pull request merged**